config/master.d/master.conf

```diff
@@ -3,7 +3,8 @@ auto_accept: True
 interface: 0.0.0.0
 event_return: mysql
 presence_events: True
-worker_threads: 10
+worker_threads: 12
+timeout: 60
```

Comment:
Disagree, discussion is from 2014 and this code was changed several times. I've tracked code history.